tests/cpp/BUILD

```
@@ -0,0 +1,40 @@
+# *******************************************************************************
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+# *******************************************************************************
+
+cc_library(
+    name = "json_stubs",
+    hdrs = glob(["stubs/score/json/**/*.h"]),
+    includes = ["stubs"],
+    visibility = [
+        "//tests/cpp:__pkg__",
+    ],
+)
+
+cc_test(
+    name = "test_kvs_cpp",
+    srcs = [
+        "test_kvs.cpp",
+        "//src/cpp:kvs_srcs",
+    ],
+    copts = [
+        "-Isrc/cpp/inc",
+    ],
+    visibility = ["//visibility:public"],
```

```suggestion
    visibility = ["//:__pkg__"],
```
limit visibility usage via test_suite at https://github.com/eclipse-score/inc_mw_per/pull/20/files#diff-3053677e4333a96b20ed76c662bca07fb2e1ce059479170139209356787da09cR51